content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -1001,9 +1001,24 @@ commands.spawn(NodeBundle {
 
 ## Winit Upgrade
 
-<div class="release-feature-authors">authors: @TODO</div>
-
-TODO.
+<div class="release-feature-authors">authors: @Thierry Berger, @mockersf</div>
+
+Through the heroic efforts of our contributors and reviewers, Bevy is [now upgraded] to use `winit 0.29`.
+[`winit`] is our windowing library: it abstracts over all of the different operating systems and input devices that end users might have,
+and provides a basically uniform API to enable a write-once, run-anywhere experience.
```

Comment:
This presently reads weird to me. These are the two solutions I've come up with to maybe make it read better. Non blocking.

```suggestion
and provides a basically uniform API to enable a write-once, run-anywhere, experience.
```
or

```suggestion
and provides a basically uniform API to enable a write-once and run-anywhere experience.
```